app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -34,6 +34,17 @@
         ariaControlsValue += "subsection_content_"+i+" ";
       }
 
+      // Create a unique prefix for each service manual topic
+      var serviceManualPrefix = "GOVUK_service_manual";
+      var serviceManualTopic = $('h1').text();
+      serviceManualTopic = serviceManualTopic.replace(/\s+/g,"_");
+      serviceManualTopic = serviceManualTopic.toLowerCase();
+      var GOVUKServiceManualTopic = serviceManualPrefix+"_"+serviceManualTopic;
```

Comment:
This variable is going to be saved in session storage, so it has been added to make it clear where these values have come from.